### DISCUSSION TITLE
Set up weekly rest days

### DISCUSSION THREAD

**Discussion by davidovic999** [mikeberger/borg_calendar#201] [General]
Hi Mike,
How do you set up weekly rest days so that repetitive tasks don't fill up these days in the calendar?
Thx

**Comment by mikeberger** [comment]
When editing an appointment, the Frequency menu has an option called "Select Days", which lets you select exactly which days of the week to repeat on.

**Comment by davidovic999** [comment]
Thanks, greetings!
